ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -137,17 +141,18 @@
                             (and service-companies (empty? companies)) service-companies
                             :else companies)
         open-link (fn [content]
-                    [:a {:style    {:color "#fff"}
+                    [:a {:style    (merge {:color "#fff"} style/result-card-header-link)
                          :href     "#"
                          :on-click #(do
                                       (.preventDefault %)
                                       (e! (ss/->ShowServiceGeoJSON
                                             (str js/document.location.protocol "//" js/document.location.host
                                                  "/export/geojson/" transport-operator-id "/" id))))}
-                     content])]
+                     content])
+        card-padding (if (< (:width app) 767) "15px" "30px")]
```

Comment:
Minor issue: feels bad to give the whole `app` state as parameter, just to determine 1 style. Could we give a style to result card or some boolean (narrow/wide)?